modules/products/jira/locals.tf

```diff
@@ -43,12 +43,11 @@ locals {
     }
   }) : yamlencode({})
 
-  # After restoring the snapshot of the Jira database, a re-index is required. To avoid interruption in the Jira
-  # service we should exclude indexing status from the health check process.
-  # For more info see: https://jira.atlassian.com/browse/JRASERVER-66970
-  ignore_index_check = var.db_snapshot_id != null ? yamlencode({
+  # By default, Jira accepts an index snapshot taken within 24hours. In order to use snapshot older than 24hours we need to update following property value.
+  # It is set to 10 years.
+  reuse_old_index_snapshot = var.shared_home_snapshot_id != null ? yamlencode({
     jira = {
-      additionalJvmArgs = ["-Dcom.atlassian.jira.status.index.check=false"]
```

Comment:
This won't be needed. as share-home will always provide index snapshot

Reply:
What if user provide the RDS snapshot but not EBS snapshot? I think is safe to leave it as it is, or to be fair add a condition to check skip index check unless both snapshots are provided.

Reply:
Shouldn't user always bring two in pair? When will be the case where user only wants to restore DB only?

Reply:
I don't think we need to officially support restoring from the partial dataset. Yes, when we are creating those datasets, we will probably need to do something like this but that is a different use case than actually using the infrastructure.

Reply:
But at the same time, I think it is possible that the shared home dataset might not contain the index snapshot. What would happen if we leave this in?